app/models/current.rb

```diff
@@ -1,2 +1,9 @@
 class Current < ActiveSupport::CurrentAttributes
+  attribute :organization
+
+  resets { Time.zone = nil }
+
+  def organization=(organization)
+    super
+  end
```

Comment:
This setter seems redundant.

Reply:
That is what I got from the docs..All we need is the organization, not any methods hanging off of it.